probert/_rtnetlinkmodule.c

```diff
@@ -291,7 +329,7 @@ listener_start(PyObject *self, PyObject* args)
 		return NULL;
 	}
 
-	r = nl_cache_mngr_add_cache(listener->mngr, listener->link_cache, _cb_link, listener);
+	r = nl_cache_mngr_add_cache_v2(listener->mngr, listener->link_cache, _cb_link, listener);
```

Comment:
That looks a little suspect, debug code or a real call ?  Geez, it is real!
https://lists.infradead.org/pipermail/libnl/2016-October/002229.html

Reply:
API versioning is gr8